.github/workflows/deploy-apidocs.yml

```
@@ -41,12 +41,9 @@ jobs:
           php-version: '8.0'
           coverage: none
 
-      - name: Download phpDocumentor v3.1
-        run: |
-          cd ./source
-          curl \
-            -L https://github.com/phpDocumentor/phpDocumentor/releases/download/v3.1.0/phpDocumentor.phar \
-            -o admin/phpDocumentor.phar
+      - name: Download latest phpDocumentor
+        working-directory: source
+        run: phive install --force-accept-unsigned --target ./admin -- phpDocumentor
```

It looks to me like the PHAR is actually signed?
```
phive install phpDocumentor
Phive 0.15.0 - Copyright (C) 2015-2021 by Arne Blankerts, Sebastian Heuer and Contributors
Downloading https://api.github.com/repos/phpdocumentor/phpdocumentor2/releases
Downloading https://github.com/phpDocumentor/phpDocumentor/releases/download/v3.1.1/phpDocumentor.phar
Downloading https://github.com/phpDocumentor/phpDocumentor/releases/download/v3.1.1/phpDocumentor.phar.asc
Downloading key 67F861C3D889C656
Trying to connect to keys.openpgp.org (37.218.245.50)
Downloading https://keys.openpgp.org/pks/lookup?op=get&options=mr&search=0x67F861C3D889C656
Successfully downloaded key.

	Fingerprint: F33A 0AF6 9AF7 A8B1 5017 DB52 6DA3 ACC4 991F FAE5

	phpDocumentor <info@phpdoc.org>

	Created: 2019-07-26

Import this key? [y|N]
```

Suggest:
```suggestion
        run: phive install --trust-gpg-keys 67F861C3D889C656 --target ./admin -- phpDocumentor
```

---

Probably throw in `--no-progress` too. I will edit suggestion.